juno/assetlist.json

```diff
@@ -506,6 +506,29 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/fanfury.png"
         },
         "coingecko_id": "fanfury"          
+      },
+      {
+        "description": "The native token cw20 for PHMN on Juno Chain",
+        "type_asset": "cw20",
+        "address": "juno1rws84uz7969aaa7pej303udhlkt3j9ca0l3egpcae98jwak9quzq8szn2l",
+        "denom_units": [
+          {
+            "denom": "cw20:juno1rws84uz7969aaa7pej303udhlkt3j9ca0l3egpcae98jwak9quzq8szn2l",
+            "exponent": 0
+          },
+          {
+            "denom": "phmn",
+            "exponent": 6
+          }
+        ],
+        "base": "cw20:juno1rws84uz7969aaa7pej303udhlkt3j9ca0l3egpcae98jwak9quzq8szn2l",
+        "name": "POSTHUMAN",
+        "display": "phmn",
+        "symbol": "PHMN",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/phmn.png"
```

Comment:
Can we get a square (not rectangular dimentions) image please?

Reply:
Not sure that I understand what do you mean, but I reuploaded slightly changes images. Hope this fixes the issue.